products/liquibase.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Liquibase
+category: framework
+tags: app java-runtime library
```

Comment:
```suggestion
tags: java-runtime
```

`library` is not an actual endoflife.date tag, and the `framework` category will be also transformed to a tag and is pretty close anyway.

`app` is also a category : this is not allowed [as per our documentation](https://endoflife.date/contribute#new-adding-a-new-product).